channeldb/graph.go

```diff
@@ -1142,7 +1142,7 @@ func (c *ChannelGraph) NewChannelEdge() *ChannelEdge {
 
 func putLightningNode(nodeBucket *bolt.Bucket, aliasBucket *bolt.Bucket, node *LightningNode) error {
 	var (
-		scratch [8]byte
+		scratch [16]byte
```

Comment:
@AndrewSamokhvalov thanks for the review and the comments! Good catch here. I made a change that required the additional bytes, and then backed that out and left this in. Fixed now.